Tennis-scheduler/src/main/java/com/tennis/tennisscheduler/services/PersonService.java

```diff
@@ -15,6 +17,8 @@
 public class PersonService {
 
     private final PersonRepository personRepository;
+    private final RoleRepository roleRepository;
+    private final PasswordEncoder bCryptPasswordEncoder;
```

Comment:
I think you have to explicitly import BCrypt to use it.

Reply:
Just check this, because i am not 100% sure

Reply:
Is it enough that i specifed in spring security config implemenantion of password encoder with bean annotation (that is BCrypt)?I'm not sure, but when i use debug it looks like it use that implemenatition.